PixelHunter1995/Content/SmurphFiles/dialog_intro.smph

```diff
@@ -0,0 +1,95 @@
+#IntroInit
+Nick: OK, so I've started the boiler and poured up the cooking oil.
+      What's the next ingredient?
```

Comment:
I think I would prefer if we write the name on every line. As it is now either we have two quite different meanings for indentation, or we disallow starting a line of dialog with "[word]:". A plausible example would be "Fact: steel contains both iron and coal."

Reply:
Done.